testcases/OpTestSTB.py

```diff
@@ -0,0 +1,623 @@
+        l_bk_not_measured = self.string_get(self.l_data,
+                                            BMC_CONST.TB_SKIPPED)
+        if l_bk_measured:
+            print BMC_CONST.BOOTKERNEL_PAR_MEASURED
+            return BMC_CONST.BOOTKERNEL_PAR_MEASURED
+        elif l_bk_not_measured:
+            print BMC_CONST.BOOTKERNEL_PAR_NOT_MEASURED
+            return BMC_CONST.BOOTKERNEL_PAR_NOT_MEASURED
+
+        l_msg =  "BOOTKERNEL partition measured details not seen"
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects CAPP verified information from OPAL
+    # messagelog and if found determines whether it is 'verified' or
+    # 'not verified'.
+    # Returns CAPP_PAR_VERIFIED for CAPP partition verified
+    # Returns CAPP_PAR_NOT_VERIFIED for CAPP partition not verified
+    # Raises OpTestError when no information found
+    #
+    def capp_verified(self):
+        l_capp_verified = self.string_get(self.l_data,
+                                          BMC_CONST.CAPP_VERIFIED)
+        l_capp_not_verified = self.string_get(self.l_data,
+                                              BMC_CONST.SB_SKIPPED)
+        if l_capp_verified:
+            print BMC_CONST.CAPP_PAR_VERIFIED
+            return BMC_CONST.CAPP_PAR_VERIFIED
+        elif l_capp_not_verified:
+            print BMC_CONST.CAPP_PAR_NOT_VERIFIED
+            return BMC_CONST.CAPP_PAR_NOT_VERIFIED
+
+        l_msg = "CAPP partition verify details not seen"
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects BOOTKERNEL verified information from OPAL
+    # messagelog and if found determines whether it is 'verified' or
+    # 'not verified'.
+    # Returns BOOTKERNEL_PAR_VERIFIED for BOOTKERNEL partition verified
+    # Returns BOOTKERNEL_PAR_NOT_VERIFIED for BOOTKERNEL partition not verified
+    # Raises OpTestError when no information found
+    #
+    def bootkernel_verified(self):
+        l_bk_verified = self.string_get(self.l_data,
+                                        BMC_CONST.BOOTKERNEL_VERIFIED)
+        l_bk_not_verified = self.string_get(self.l_data,
+                                            BMC_CONST.SB_SKIPPED)
+        if l_bk_verified:
+            print BMC_CONST.BOOTKERNEL_PAR_VERIFIED
+            return BMC_CONST.BOOTKERNEL_PAR_VERIFIED
+        elif l_bk_not_verified:
+            print BMC_CONST.BOOTKERNEL_PAR_NOT_VERIFIED
+            return BMC_CONST.BOOTKERNEL_PAR_NOT_VERIFIED
+
+        l_msg = "BOOTKERNEL partition verify details not seen"
+        raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects CAPP loaded information from OPAL
+    # messagelog and if found determines whether it is 'loaded' or
+    # 'not loaded'.
+    # Returns CAPP_PAR_LOADED for CAPP partition loaded
+    # Returns CAPP_PAR_NOT_LOADED for CAPP partition not loaded
+    # Raises OpTestError when no information found
+    #
+    def capp_loaded(self):
```

Comment:
Change function name to opal_get_capp_loaded_info. and just return True/ False for Loaded/not-loaded cases.